utils/DegenEmbed.py

```diff
@@ -1,25 +1,36 @@
 import discord
+from datetime import datetime
 
 
-class DegenEmbed:
-    def __init__(self, title, description, logo="https://avatars.githubusercontent.com/u/1737241?v=4", color=0x1f6e9e):
-        self.embed_title = title
-        self.embed_description = description
-        self.embed_logo = logo
-        self.embed_color = color
-        self.embed = discord.Embed(title=self.embed_title, url="https://www.seattledegens.wtf", description=self.embed_description,
-                                   color=self.embed_color)
+class DegenEmbed(discord.Embed):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
 
-# Need to_dict for some embed internals.
-    def to_dict(self):
-        return self.embed.to_dict()
-
-    def create(self, thumbnail, title=None):
-        # I don't really like the author field atm.
-        ## self.embed.set_author(name=f"Get Fucked", icon_url=self.embed_logo)
-        self.embed.set_thumbnail(url=thumbnail)
-        self.embed_title = title
-        self.embed.set_footer(text="Made by xDDYx's darkest thoughts.")
+    def create(self, thumbnail=None, title=None):
+        if thumbnail:
+            self.set_thumbnail(url=thumbnail)
+        if title:
+            self.title = title
+        self.set_footer(text="Made by xDDYx's darkest thoughts.")
 
     def add_field(self, name, value, inline=False):
-        self.embed.add_field(name=name, value=value, inline=inline)
\ No newline at end of file
+        super().add_field(name=name, value=value, inline=inline)
+
+    def construct_full_embed(cls, games: object, title: object, color: object) -> object:
+        newembed = DegenEmbed(title=title, color=color)
+        try:
+            newembed.create(games[0].team['logo_url'], title)
+        except Exception:
+            newembed.create("https://krakenhockeyleague.com/hockey/images/teamlogos100/Degens.png", title)
+
+        for game in games:
+            if game.gametime <= datetime.now():
+                newembed.add_field(f'{game.away_team}: {game.away_score} '
+                                   f'{game.home_team}: {game.home_score}',
+                                   f'{game.gametime.strftime('%B %d')}',
+                                   inline=False)
+            else:
+                newembed.add_field(f'{game.away_team} Vs. {game.home_team} ',
+                                   f'{game.gametime} @ {game.location}', inline=False)
```

Comment:
I like this format, but the gametime needs a strftime to make it look better

Reply:
Yeah I agree but I feel like that should be done upstream and not here.

Reply:
Coffee thought this morning we could replace the timestamps with links to the VOD if the game is past - since people seem to be using this bot as a way to reference their own schedules.

Reply:
yeah i think we should add a link to the khl page or w/e it is as well